lib/gateway.js

```diff
@@ -281,6 +281,22 @@ async function buildGateway (gatewayOpts, app) {
     schema,
     serviceMap,
     entityResolversFactory: factory,
+    pollingInterval: gatewayOpts.pollingInterval,
+    async refresh () {
+      for (const service of Object.values(serviceMap)) {
+        try {
+          await service.refresh()
+        } catch (err) {
+          continue
+        }
+      }
```

Comment:
Can we refresh those in parallel with a `Promise.allSettled()`?

Reply:
I have updated `.refresh()` method for getting information from other services in parallel.

Reply:
`Promise.allSettled()` works only with Node.js 12.9.0+